helix-front/angular.json

```diff
@@ -5,28 +5,28 @@
   "projects": {
     "helix-front": {
       "root": "",
-      "sourceRoot": "client",
+      "sourceRoot": "src",
```

Comment:
The majority of the diff is related to this change, which was required by storybook.

Error message:

```bash
The client/assets asset path must start with the project source root.
```

https://github.com/apache/helix/issues/2203#issuecomment-1236168591